src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in

```diff
@@ -13,8 +13,13 @@
     <BuildServerSideRenderer>false</BuildServerSideRenderer>
     <!-- aspnet/BuildTools#662 Don't police what version of NetCoreApp we use -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
```

Comment:
We should probably remove these 3 lines from templates. They are workarounds for tests, but they are not content we actually want in the .csproj files we generate for users.